build/release/post-release.sh

```diff
@@ -33,7 +33,9 @@ cd -
 npm run release:dist $1 $2
 cd $dist
 git add -A
-npm version $1
+git commit -m "Release: $1"
+# -s to sign and annotate tag (recommended for releases)
+git tag -s $1 -m "Release: $1"
```

Comment:
`npm version` wasn't quite right. Aside from the need to commit first anyway, the correct version is already set at this point in the package.json. But more importantly, there's no `.npmrc` in the jquery-dist repo, so the tag name would be wrong.

Reply:
The correct version is needed not just in `package.json`, but also in the lockfile - and there, depending on the version, even twice! For this reason, it’s be safer to rely on npm version; the format can be tweaked, I believe.

Reply:
This is for jquery-dist. There's no lockfile.

Reply:
Ah, right. There are no dependencies there so it wouldn’t be useful to commit one.